### PR TITLE
Ria 3731 remove representation

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3731](https://tools.hmcts.net/jira/browse/RIA-3731)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/pages/notice-of-change.page.ts [hunk 1]
```diff
@@ -160,6 +160,10 @@ export class NoticeOfChangePage {
         console.log('\n\tLatest case id : ' + this.latestCaseId + '\n')
     }
 
+    async goToLatestCase(shortWait = false) {
+        browser.driver.get('http://localhost:3002/cases/case-details/' + this.noticeOfChangeCaseId);
```
Comment (makowalski) [inline e2e/pages/notice-of-change.page.ts:164]: Ideally we should use below variable:
```
  CcdWebUrl: process.env.TEST_E2E_URL_WEB || 'https://manage-case.aat.platform.hmcts.net',

```

Comment (bradsorour) [reply e2e/pages/notice-of-change.page.ts]: CcdWebUrl now replaces localhost.

**Review by makowalski — APPROVED** [review]
LGTM

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**